client/components/widget/data_viz/gviz/chart-config-directive.html

```diff
@@ -26,39 +26,112 @@
                  ng-model="ngModel.layout.columnspan">
         </div>
       </div>
-
+    </div>      
+      
+    <div class="sidebar-group">
+      <div class="sidebar-group-title">Chart</div>
+  
       <div class="sidebar-item">
         <div class="sidebar-item-label">chart type</div>
```

Comment:
The spacing is admittedly poor, but this PR is part of a larger effort moving us to the sidebar UX.  As padding/margin/title considerations will change in a few PR's, I'll leave the visual tweaks till then, and address it across all sidebars.